### ISSUE TITLE
mob head set to 0 and they still target me from far away

### ISSUE THREAD

**Issue by ararwolff** [Laike-Endaril/Dynamic-Stealth#136]
NOTE: I also have a mod that makes mobs see farther distances, but even with mob head multiplier set to 0 they shouldn't even see me at all

# When an entity (including a player) is wearing a mob head, mobs of that type have reduced chance to realize they're a target
                    # 
                    # If set to 1, there is no effect
                    # 
                    # If set to 0, mobs of the mob head type cannot notice entities wearing their heads
                    # Min: 0.0
                    # Max: 1.0
                    D:"Mob Head Multiplier"=0

**Comment by ararwolff** [comment]
I should add that I have 'zombie awareness' mod and 'monster swarm' mod

**Comment by Laike-Endaril** [comment]
I was on a years-long hiatus when this was posted, and I assume you don't care at this point, but just in case this helps anyone...

I can think of quite a few reasons this might not be working, and all of them involve mod compatibility (the system should work fine without other mods).

That being said, it would be a good system to improve on / make more compatible, and I had plans to do so that I never got around to.  I might re-continue this project, so I'll leave this open.

**Comment by Laike-Endaril** [comment]
Note (mostly to self): what I was planning on doing was a better "disguise" system in DS.  This would actually not solve the issue the OP had, as that is some kind of compat issue with an unknown mod, and I don't have enough information to fix that.

**Comment by Laike-Endaril** [comment]
Right, an improved disguise system is already in my internal to-do list, the actual original issue here can't be addressed without more info, and this is super old.  I'm closing this one for now.  If someone has a similar issue, reply here and I can re-open this (make sure to include what mod is causing a conflict if so).
